docs/log-management/index.md

```diff
@@ -44,8 +34,8 @@ presets:
 config:
   exporters:
     otlp:
-      # Kubernetes endpoint as here or directly to your local instance localhost:7281
-      endpoint: quickwit-indexer.qw-tutorial.svc.cluster.local:7281
+      # Local instance localhost:7281
```

Comment:
When first reading this I'm not really sure what this means, is the config setup for k8s but this comment saying you can use localhost if running locally?

Reply:
I need to be clearer here indeed.

Reply:
Could be any host, on k8s you will want a url of the form `{quickwit-indexer-service-name}.{namespace}.svc.cluster.local:7281`